docker_images/mbed-os-env/Dockerfile

```diff
@@ -0,0 +1,103 @@
+# ------------------------------------------------------------------------------
+# Pull base image
+FROM ubuntu:20.04
+
+# ------------------------------------------------------------------------------
+# Arguments
+ARG WORKDIR=/root
+
+# ------------------------------------------------------------------------------
+# Install tools via apt
+RUN ls
+
+ADD requirements.txt .
+ENV DEBIAN_FRONTEND=noninteractive
+RUN set -x \
+    && apt -y update \
+    && apt -y install \
+    git \
+    wget \
+    python3 \
+    python3-dev \
+    python3-setuptools \
+    python3-usb \
+    python3-pip \
+    software-properties-common \
+    build-essential \
+    astyle \
+    mercurial \
+    ninja-build \
+    libssl-dev \
+    cargo \
+    && apt clean && rm -rf /var/lib/apt/lists \
+    && update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1 \
+    && : # last line
+
+# Cmake (Mbed OS requires >=3.19.0-rc3 version which is not available in Ubuntu 20.04 repository)
+RUN set -x \
```

Comment:
Kitware is proposing install of CMake through pip: https://pypi.org/project/cmake/ it might be more portable to use this. However I don't know why versions 3.19 have been skipped.

Reply:
Thanks, will have a look. Hopefully they have aarch64 support too...